changelog.d/9265.bugfix

```diff
@@ -0,0 +1 @@
+Prevent password hashes from getting dropped if a client failed threepid validation during a User Interactive Auth stage.
```

Comment:
Does it make sense to mention that this removes a workaround for Element < *some number*?

Reply:
Or at least "for old Riots"?

Reply:
Oh, indeed yes. The fix was added in Riot v0.7.4. I'll add that to the changelog.